test/decoders/test_decoders.py

```diff
@@ -939,11 +940,18 @@ def get_some_frames(decoder):
 
 
 class TestAudioDecoder:
-    def test_metadata(self):
-        decoder = AudioDecoder(NASA_VIDEO.path)
+    @pytest.mark.parametrize("asset", (NASA_AUDIO, NASA_AUDIO_MP3))
+    def test_metadata(self, asset):
+        decoder = AudioDecoder(asset.path)
         assert isinstance(decoder.metadata, AudioStreamMetadata)
 
-        assert decoder.stream_index == decoder.metadata.stream_index == 4
-        assert decoder.metadata.duration_seconds == pytest.approx(13.056)
-        assert decoder.metadata.sample_rate == 16_000
-        assert decoder.metadata.num_channels == 2
+        assert (
+            decoder.stream_index
+            == decoder.metadata.stream_index
+            == asset.default_stream_index
+        )
+        assert decoder.metadata.duration_seconds == pytest.approx(
+            asset.duration_seconds
+        )
+        assert decoder.metadata.sample_rate == asset.sample_rate
+        assert decoder.metadata.num_channels == asset.num_channels
```

Comment:
Changes here are mostly a drive-by. I'm extending this one because I removed `test_audio_get_json_metadata` which was outdated.